mon-pix/tests/acceptance/start-campaigns-test.js

```diff
@@ -122,6 +128,70 @@ describe('Acceptance | Campaigns | Start Campaigns', function() {
               expect(currentURL()).to.equal(`/campagnes/${campaignCode}/presentation`);
             });
 
+            it('should not alter inputs(username,password,email) when email already exists ', async function() {
+
+              //given
+              this.server.put('student-user-associations/possibilities', () => {
+
+                const studentFoundWithUsernameGenerated = {
+                  'data': {
+                    'attributes': {
+                      'last-name': 'last',
+                      'first-name': 'first',
+                      'birthdate': '2010-10-10',
+                      'campaign-code': 'RESTRICTD',
+                      'username': 'first.last1010'
+                    }, 'type': 'student-user-associations'
+                  }
+                };
+
+                return _overrideResponseApi(200,studentFoundWithUsernameGenerated);
+              });
+
+              this.server.post('student-dependent-users', () => {
+
+                const stubErrorInvalidDataEmailAlreadyExist = {
+                  'errors': [{
+                    'status': '422',
+                    'title': 'Invalid data attribute "email"',
+                    'detail': 'Cette adresse e-mail est déjà enregistrée, connectez-vous.',
+                    'source': { 'pointer': '/data/attributes/email' }
+                  }]
+                };
+
+                return _overrideResponseApi(422,stubErrorInvalidDataEmailAlreadyExist);
+              });
+
+              await visitWithAbortedTransition(`/campagnes/${campaignCode}`);
+
+              // when
+              await fillIn('#firstName', 'JeanPrescrit');
+              await fillIn('#lastName', 'Campagne');
+              await fillIn('#dayOfBirth', '10');
+              await fillIn('#monthOfBirth', '12');
+              await fillIn('#yearOfBirth', '2000');
+
+              await click('#submit-search');
+              //switch to email login mode
+              await click('.pix-toggle__off');
+
+              await fillIn('#email', 'JeanPrescrit.Campagne@example.net');
+              await fillIn('#password', 'pix123');
+              await click('#submit-registration');
+
+              // then
+              expect(currentURL()).to.equal(`/campagnes/${campaignCode}/identification`);
+              expect(find('#firstName').value).to.equal('JeanPrescrit');
+              expect(find('#email').value).to.equal('JeanPrescrit.Campagne@example.net');
+              expect(find('#password').value).to.equal('pix123');
+
+              //switch to username login mode
```

Comment:
// go to username-based authentication window

Reply:
C'est plus un switch entre deux modes de login.